com.github.G_Ray.PikaTorrent.yml

```diff
@@ -0,0 +1,45 @@
+app-id: com.github.G_Ray.PikaTorrent
```

Comment:
it should be `io.github`

Reply:
Thanks, actually I control pikatorrent.com so I updated the app id to `com.pikatorrent.PikaTorrent`